### PR TITLE
Experience/10910-Website-Redesign---Developer-Resources

### LINKED ISSUE #10910: Website Redesign - Developer Resources
As a Developer User,
I want to visit the ReportStream **Developer Resources** Page,
So that I can become informed about the technical details of interacting with ReportStream.

Designs
- Figma Link: https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design

Content Updates
- [See Figma Link](https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design)

Page Title
- "CDC ReportStream - Developer Resources" 

Analytics
- All links
- Page Load

Storybook updates
- TBD

Verification Steps
- Signed off in Staging by Product / Design
- Signed off in Production by Product / Design / Engineering

Acceptance Criteria:
- [ ] Analytics / Metrics Required 
- [ ] Storybook update required
- [ ] Unit Tests
- [ ] End to End Tests
- [ ] Manual Testing Steps

Dependencies:
- Designs - https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design
- Platform API Endpoints - N/A
- Links - TBD

Additional Details:
- The deployment strategy will follow the general deployment strategy for the Website Redesign EPIC. We will deliver this page in isolation from the other pages. 
- Conversion to MDX will be determined by the Engineer owning this ticket

### PR DESCRIPTION
**Pull request by jpandersen87**
Fixes #10910

This PR implements the developer resources landing page.

### CODE REVIEW

**Review by KeithNava — APPROVED** [review]

**Review by etanb — APPROVED** [review]

File: frontend-react/src/content/developer-resources/landing-page.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (penny-lischer) [inline frontend-react/src/content/developer-resources/landing-page.mdx:1]: can we rename this to developer-resources-index to keep consistent with the other landing pages? or rename the other ones?

File: frontend-react/src/content/developer-resources/landing-page.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - title: Release notes
+      href: /release-notes
+      content: View monthly updates on new features, enhancements, and fixes.
+---
+<HeroWrapper>
+<section className="usa-section grid-container">
+<CardGroup>
+        <Card gridLayout={{tablet: {col: 4}}}>
+            <CardHeader>
+                ## [API Guide](/api-guide)
```
Comment (penny-lischer) [inline frontend-react/src/content/developer-resources/landing-page.mdx:19]: This should be "ReportStream API" instead of "API Guide"

**Review by jillian-hammer — APPROVED** [review]
hi @jpandersen87 thank you, this looks great! the only thing I caught was:

- adjusting page header to sentence case ("Developer Resources" to "Developer resources")

i tried to edit it on my own, so let me know if i did it correctly! 🤞

**Comment by jpandersen87** [comment]
> hi @jpandersen87 thank you, this looks great! the only thing I caught was:
> 
> * adjusting page header to sentence case ("Developer Resources" to "Developer resources")
> 
> i tried to edit it on my own, so let me know if i did it correctly! 🤞

You did! 🎉

File: frontend-react/src/content/developer-resources/landing-page.mdx (hunk 2 above)
Comment (jpandersen87) [reply frontend-react/src/content/developer-resources/landing-page.mdx:19]: What you're saying does not reflect the current figma sheet.

**Comment by audreykwr** [comment]
- [ ] my b - need to uncapitalize "guide" in API Guide
- [ ] API guide link is not working
- [ ] release notes link not working

**Review by penny-lischer — APPROVED** [review]
LGTM

**Review by yudonoodles — APPROVED** [review]

**Review by audreykwr — APPROVED** [review]

**Pull request merged**